benchopt/tests/test_installcmd_solver.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+
+from benchopt.cli.main import install
+from benchopt.utils.temp_benchmark import temp_benchmark
+
+
+def test_invalid_install_cmd():
+    # Solver with an invalid install command
+    invalid_solver = """
+    from benchopt import BaseSolver
```

Comment:
To force install, you need to mimic the fact that this solver is not working yet:
```suggestion
    from benchopt import BaseSolver, safe_import_context
    
    with safe_import_context() as import_ctx:
        raise ImportError()
    
```